department-of-veterans-affairs/vets-website
# 78597 Update content about single direct deposit account requirement

## Summary
Component and content updates on the `payment-information` page when adding/editing bank account information. 

There are two views: 1) Payment Information and 2) Add new bank account. These views have different components/content depending on pre-fill vs no pre-fill and updated vs not updated.

There are a total of 5 changes. Refer to the manual verification steps and attached pdf with screenshots.

- _(Which team do you work for, does your team own the maintenance of this component?)_
@department-of-veterans-affairs/dbex-trex 

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#78597

## Testing done
- Manual testing and updated unit tests

**Verification steps**
1. No prefill
Pre-condition: find test user that has no bank account to pre-fill, or otherwise suppress prefill
1.1. Sign in and start a new claim
1.2. Fill out the form until you get to the `payment-information` page, which is in the last chapter
1.3 **Verify** content **A** displays on the 'Add new bank account' view
1.4 Fill out all required account information fields and click 'Save' button
1.5 **Verify** 'Payment information' view displays with content **F**
1.6 Go back and remove all newly added bank info and click 'Save' button
1.7 **Verify (regression)** empty Payment information view displays with no alerts

2. Pre-fill
Pre-condition: find test user that has bank account info to pre-fill
2.1 Sign in and start a new claim
2.2 Fill out the form until you get to the `payment-information` page, which is in the last chapter
2.3 **Verify** 'Payment information' view displays with content **E**
2.4 Click the 'New account' button
2.5 **Verify** 'Add new bank account' view displays with content **A**
2.6 Fill out all required account information fields and click 'Save' button
2.7 **Verify** 'Payment information' view displays with content **D** and that VA profile link to `/profile/direct-deposit` opens in new tab

## Screenshots
[payment-information PR notes.pdf](https://github.com/department-of-veterans-affairs/vets-website/files/14765882/payment-information.PR.notes.pdf)

## What areas of the site does it impact?
526EZ

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
